.github/workflows/python.yml

```diff
@@ -80,7 +80,7 @@ jobs:
         run: |
           mkdir proto-check
           poetry run python -m grpc_tools.protoc -I../../../protos --python_out=./proto-check --pyi_out=./proto-check --grpc_python_out=./proto-check ../../../protos/script.proto
-          ../../../scripts/fix_grpc_imports.sh ./proto-check
+          ./fix_grpc_imports.sh ./proto-check
```

Comment:
Why not use `sdks/python/generate.sh` here?

Reply:
we can't do that since it's another directory

Reply:
Then take the directory as an argument in generate.sh like fix_grpc_imports.sh does.